package.json

```diff
@@ -38,7 +38,7 @@
     "noflo-objects": "~0.2.0",
     "noflo-polymer": "~0.2.0",
     "noflo-routers": "~0.2.2",
-    "noflo-runtime": "~0.3.13",
+    "noflo-runtime": "~0.3.14",
```

Comment:
Can we move the components from noflo-runtime up here soon? They seem very tightly connected...